episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+In this tutorial we are going to learn how to use the `{cfr}` package to calculate and adjust a CFR estimation using [delay distributions](../learners/reference.md#delaydist) from `{epiparameter}` or elsewhere, based on the methods developed by [Nishiura et al., 2009](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006852), also, how we can reuse `{cfr}` functions for more severity measurements.
+
+We’ll use the pipe `%>%` to connect some of their functions, so let’s also call to the `{tidyverse}` package:
+
+```{r,message=FALSE,warning=FALSE}
+library(cfr)
+library(epiparameter)
+library(tidyverse)
+```
+
+## Naive CFR
+
+We measure disease severity in terms of case fatality ratio (CFR) --which is given by the ratio of the cumulative number of deaths $D_{t}$ to the cumulative number of confirmed cases $C_{t}$ at a certain time $t$. We can refer to the _naive CFR_ (also crude or biased CFR, $b_{t}$):
+
+$$ b_{t} =  \frac{D_{t}}{C_{t}} $$
+
+<!-- total number of deaths $D$ divided by total number of confirmed cases $C$ -->
+
+:::::::::::::: callout
+
+### What are data sources to estimate CFR?
+
+[Verity et al., 2020](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30243-7/fulltext) summaries the spectrum of COVID-19 cases.
+
+![Spectrum of COVID-19 cases. The CFR aims to estimate the proportion of Deaths among confirmed cases in an epidemic.](fig/cfr-spectrum-cases-covid19.jpg)
+
+- At the top of the pyramid, those meeting the WHO case criteria for **severe** or critical cases will likely be identified in the hospital setting, presenting with atypical viral pneumonia. These cases will have been identified in mainland China and among those categorised internationally as local transmission. 
+- Many more cases are likely to be **symptomatic** (i.e., with fever, cough, or myalgia) but might not require hospitalisation. These cases will have been identified through links to international travel to high-risk areas and through contact-tracing of contacts of confirmed cases. They might also be identified through population surveillance of, for example, influenza-like illness. 
+- The bottom part of the pyramid represents **mild** (and possibly **asymptomatic**) cases. These cases might be identified through contact tracing and subsequently via serological testing.
+
+::::::::::::::
+
+To calculate the naive CFR, the `{cfr}` package requires a data frame with three columns named:
+
+- `date`
+- `cases`
+- `deaths`
```

Comment:
An important point here is that {cfr} works on aggregated incidence data. I have not seen this mentioned in this episode, but the use of incidence data is shown in earlier episodes -- may be worth adding an explanation to distinguish this from linelist data.

Reply:
I relocated the callout below and added this information more explicitly using EpiNow2 aggregated data and outbreak linelist data https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/f4d6942ea8f27bbbd24ce808f53cb9a4ea4388dd